esp32c6-lp/svd/patches/esp32c6-lp.yaml

```diff
@@ -1 +1,10 @@
 _svd: "../esp32c6-lp.base.svd"
+
+LP_I2C0:
+  _strip: "I2C_"
+  "*":
+    _strip: "I2C_"
+  DATA:
+    _modify:
+      FIFO_RDATA:
+        access: read-write
```

Comment:
Is FIFO rddata actually read-write? Or is it readonly?

Reply:
It's read-write, we are using it in HP i2c as well: https://github.com/esp-rs/esp-hal/blob/main/esp-hal/src/i2c.rs#L1555-L1558
The name is very confusing :D

Reply:
Haha thanks!